autode/wrappers/ORCA.py

```diff
@@ -537,5 +445,112 @@
         # Hessians printed in Ha/a0^2, so convert to base Ha/Å^2
         return np.array(hessian, dtype='f8') / Constants.a0_to_ang**2
 
+    @work_in_tmp_dir(filenames_to_copy=[], kept_file_exts=[])
+    def _get_version_no_output(self) -> str:
+        """
+        Get the version of ORCA without an existing output file
+        """
+
+        try:
+            run_external(params=[self.path, '-h'], output_filename='tmp')
+            line = next(l for l in open('tmp', 'r') if 'Program Version' in l)
+            return line.split()[2]
+
+        except (OSError, IOError, StopIteration):
+            return '???'
+
+    def get_keywords(self, calc_input, molecule):
+        """Modify the keywords for this calculation with the solvent + fix for
+        single atom optimisation calls"""
+
+        new_keywords = []
+
+        for keyword in calc_input.keywords.copy():
+            if 'opt' in keyword.lower() and molecule.n_atoms == 1:
+                logger.warning('Can\'t optimise a single atom')
+                continue
+
+            if isinstance(keyword, kws.ECP) and keyword.orca is None:
+                # Use the default specification for applying ECPs
+                continue
+
+            if isinstance(keyword, kws.MaxOptCycles):
+                continue  # Set in print_num_optimisation_steps
+
+            if isinstance(keyword, kws.Keyword):
+                new_keywords.append(keyword.orca)
+
+            else:
+                new_keywords.append(str(keyword))
+
+        if molecule.solvent is not None:
+            self.add_solvent_keyword(molecule, new_keywords)
+
+        # Sort the keywords with all the items with newlines at the end, so
```

Comment:
is it guarenteed that there will be exactly one keyword with no newline (or does it not matter?)

Reply:
Nope, but I don't think it should matter